src/components/Main/Team/Team.tsx

```diff
@@ -36,12 +38,48 @@ function Team(props: Props) {
     fetchTeam();
   }, []);
 
+  var count = 0;
+  var empty = {
+    name:"",
+    id:"empty",
+    position:"",
+    academicYear:"",
+    headshot: {
+      url:"",
+    }
+  }
+  console.log(arrays.length);
+  while(team.length%3 !== 0) {
+    team.push(empty);
+  }
+  for(let i = 0; i<team.length; i+=3) {
+    arrays[count] = [team[i],team[i+1],team[i+2]];
+    count++;
+  }
+
   return (
     <div id={props.id}>
-      <h2>The best work...</h2>
-      <p>Our leadership team...</p>
-      {/* iterate through team for the profile. hint: console.log() to see what the data structure looks like and how you can use it */}
-      <Profile name="Emily" img="" position="President" />
+      <Row className="m-3 d-flex justify-content-center">
+        <Col lg={6} md={8} sm={10}>
+          <h2 className="text-peacock font-weight-bold">The best work is produced when diverse voices help create it.</h2>
+          <p className="text-dark-grey pt-3">Our leadership team works together to disrupt the image of stereotypical programmer. Meet our wave-makers!</p>
+        </Col>
+      </Row>
+      {arrays.map(mem => 
+        <div className={isMobile ? "container pt-1 pb-1" : "container pt-3 pb-3"}>
+          <Row className="pl-4 justify-content-center">
+            <Col xs={4} lg={3} md={4} sm={5}>
```

Comment:
why these numbers? it should be 3 in a row for desktop, 2 in a row for mobile

```
<div className={isMobile ? "container pt-1 pb-1" : "container pt-3 pb-3"}>
        <Row className="pl-4 justify-content-center">
          {arrays.map(mem => 
            <Col xs={6} md={4}>
              <Profile name={mem[0].name} img={mem[0].headshot.url} position={mem[0].position}></Profile>
            </Col>
          )}
        </Row>
      </div>
```

the above code isn't perfect, but shows how you can simplify your code